src/coinbase/address.ts

```diff
@@ -1,63 +1,19 @@
-import { Decimal } from "decimal.js";
-import { ethers } from "ethers";
-import { Address as AddressModel } from "../client";
-import { Asset } from "./asset";
-import { Balance } from "./balance";
-import { BalanceMap } from "./balance_map";
-import { Coinbase } from "./coinbase";
-import { ArgumentError, InternalError } from "./errors";
-import { FaucetTransaction } from "./faucet_transaction";
-import { Trade } from "./trade";
-import { Transfer } from "./transfer";
-import { Amount, Destination, TransferStatus } from "./types";
-import { delay } from "./utils";
-import { Wallet as WalletClass } from "./wallet";
-
 /**
  * A representation of a blockchain address, which is a user-controlled account on a network.
  */
 export class Address {
-  private model: AddressModel;
-  private key?: ethers.Wallet;
+  protected _networkId: string;
+  protected _id: string;
 
   /**
```

Comment:
We can follow the same pattern when the methods are not implemented: https://github.com/coinbase/coinbase-sdk-ruby/blob/v0.0.9/lib/coinbase/address.rb#L38